.github/workflows/validate-cg-manifest.sh

```diff
@@ -23,6 +23,8 @@ ignore_multiple_sources=" \
 
 # List of ignored specs due to no source tarball to scan.
 ignore_no_source_tarball=" \
+  azurelinux-repos \
```

Comment:
I'd leave the package names the way they are, since `mariner-release` and `mariner-repos` have already been re-named to start with `azurelinux` in other PRs.